src/routes/user/post.route.js

```diff
@@ -1,27 +1,8 @@
-const User = require('../../entities/user')
-
 module.exports.post = {
 	method: 'POST',
 	path: '/test',
-	schema: {
-		body: {
-			type: 'object',
-			additionalProperties: false,
-			required: [
-				'username'
-			],
-			properties: {
-				username: {
-					type: 'string'
-				}
-			}
-		}
-	},
-	handler: async (req, res, next) => {
-		const { username } = req.body
-
-		const user = await User.findOne({ username })
-		if (!user) return next(new StatusError('User', 404))
-		res.json(user)
+	requiresAuth: true,
+	handler: async (req, res, _next) => {
+		res.status(200).send(req.user)
```

Comment:
this is a good example now of authentication. when `requiresAuth === true`, passport attaches the user object to `req`. so we can just retrieve it from there and do whatever